internal/mcfg/templates/ignition.gotmpl

```diff
@@ -0,0 +1,77 @@
+ignition:
```

Comment:
Can we somehow reuse `pkg/mcproducer/templates/machine-config.gotmpl`? Do we have to redefine a template?